### DISCUSSION TITLE
Version 1.3 vs 1.4 pre-release?

### DISCUSSION THREAD

**Discussion by ChrisJefferson** [quarto-dev/quarto-cli#7211] [Q&A]
### Description

I notice there is a v1.3, and v1.4 pre-release. I was just wondering which is the most commonly used? I know in some software most people use the up to date pre-release, while in other software pre-releases can be quite unstable.

**Comment by mcanouil** [comment]
It really depends in which context you want to use Quarto.
I personally use the development version (not the pre-release) for everything but my main job.
In my main work, for reporting, we are using the stable 1.3 version, but for our internal documentation we are using the latest pre-release version.

**Comment by cderv** [comment]
> I know in some software most people use the up to date pre-release, while in other software pre-releases can be quite unstable.

For us, pre-release a bit a more changing that a version build before a next release. For that we have `release-canditate`. 

So I would say for Quarto: 

* pre-release: They are daily builds including new fixes and new features. Builds passes all the tests and are working version. However, there is no guarantee about no breakage between pre-release, and no breakage compare with release. So it is a way to get the latest available stuff easily (and help test the software). When using this, it needs to be updated daily by the user to get a good user experience. 
* release candidate: We build those right before we are ready to do a new stable version. Usually this means that new feature are considered stable enough to be release, and release candidate are there for everyone to test and send us feedback before release. With a release candidate, there won't be any breakage, only bugfix could be added. 
* Stable release: This is when everything is ready, new stable is out. After that there could be patch on the stable release, but everything else will happen in a new version and starting new pre-release build. There won't be lots of update and this is the stable experience. 

What you want to use depends and what you are willing to support as changes. Hope it helps clarifies.

**Reply by ChrisJefferson** [reply]
Thanks both for your replies! I'm just starting a blog, so I think I'll start with the stable release.

**Reply by kompre** [reply]
When do you think version 1.4 will hit the stable release phase? 

I've read the highlights and there are some interesting features that I would like to try (inline python code execution and new crossref), but also I would prefer to stay in the stable release.

**Reply by mcanouil** [reply]
Later this year maybe a bit later.
